frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -35,7 +35,8 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           isInline
         />
         <p className="co-legend co-required ceph-ocs-desc__legend">
-          Select at least 3 nodes in different failure domains you wish to use.
+          Select at least 3 nodes in different failure domains with a minimum requirements of 16
+          cores and 64 GiB of RAM per node.
```

Comment:
```suggestion
          CPU cores and 64 GiB of RAM per node.
```

Reply:
@cloudbehl This message is as per the @yuvalgalanti suggestion. Also `CPU` keyword was not present even in the earlier warning message.